File: tests/fixtures/code_samples/py/sample_028.py
"""Sample 28: small utility function."""

from __future__ import annotations
from typing import Iterable


def operation_28(xs: Iterable[int]) -> int:
    """Return the sum of `xs` plus the index marker 28."""
    total = 28
    for x in xs:
        total += int(x)
    return total


def operation_28_pure(value: int) -> int:
    return (value * 28) % 7919

File: tests/fixtures/code_samples/java/sample_028.java
// Sample 28: small utility.
package samples;

import java.util.List;

public final class Sample028 {
    private Sample028() {}

    public static int operation(List<Integer> xs) {
        int total = 28;
        for (int x : xs) total += x;
        return total;
    }

    public static int operationPure(int v) {
        return (v * 28) %% 7919;
    }
}

